Package cannot be installed due to `husky` on `preinstall` script

Installing the package starting on `0.1.1` breaks both in npm and in yarn:

npm:

```
➜  npm i @zendeskgarden/ckeditor
npm ERR! code 127
npm ERR! path /Users/username/test/node_modules/@zendeskgarden/ckeditor
npm ERR! command failed
npm ERR! command sh -c -- husky install
npm ERR! sh: husky: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/username/.npm/_logs/2023-06-20T08_08_49_617Z-debug-0.log
``` 

```
➜  yarn add @zendeskgarden/ckeditor
yarn add v1.22.19
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
error /Users/username/test/node_modules/@zendeskgarden/ckeditor: Command failed.
Exit code: 127
Command: husky install
Arguments:
Directory: /Users/username/test/node_modules/@zendeskgarden/ckeditor
Output:
/bin/sh: husky: command not found
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

--- a/package.json
+++ b/package.json
@@ -18,12 +18,11 @@
     "build:storybook": "build-storybook",
     "format": "prettier-package-json --write && yarn format:all --write",
     "format:all": "prettier --loglevel warn '**/*.{js,ts,json,md}' '.storybook/**/*.{js,ts,json,md}' '!CHANGELOG.md' '!dist/**'",
-    "postinstall": "husky install",
     "lint": "yarn lint:js && yarn lint:css && yarn lint:md",
     "lint:css": "stylelint 'src/**/*.css'",
     "lint:js": "eslint src/ .storybook/ --ext js,ts --max-warnings 0",
     "lint:md": "markdownlint README.md src/**/*.md",
-    "prepare": "yarn build",
+    "prepare": "yarn build && husky install",
     "start": "start-storybook -p 6006",
     "tag": "[ `git rev-parse --abbrev-ref HEAD` = 'main' ] && standard-version --no-verify",
     "test": "prettier-package-json --list-different && yarn format:all --check && yarn lint"

